Options.py

```diff
@@ -587,7 +587,7 @@ def verify_keys(cls, data):
                 raise Exception(f"Found unexpected key {', '.join(extra)} in {cls}. "
                                 f"Allowed keys: {cls.valid_keys}.")
 
-    def verify(self, world, plando_options):
+    def verify(self, world, plando_options, player_name):
```

Comment:
The lowest implementation should absolutely have typing information in the signature for auto-complete to be nice.